open-sphere-base/core/src/main/java/io/opensphere/core/units/angle/MGRS.java

```diff
@@ -0,0 +1,58 @@
+package io.opensphere.core.units.angle;
+
+public class MGRS extends Coordinates
+
+{
+
+    /**
+     * The serial ID.
+     */
+    private static final long serialVersionUID = 8396025878700604825L;
+
+    /**
+     * Construct with a specified value in degrees.
+     *
+     * @param magnitude degrees
+     */
+    public MGRS(double magnitude)
+    {
+        super(magnitude);
+    }
+
+    @Override
+    public String getLongLabel()
+    {
+        return "MGRS";
+    }
+
+    @Override
+    public String getShortLabel()
+    {
+        return "MGRS";
+    }
+
+    @Override
+    public String toShortLabelString()
+    {
+        return "";
```

Comment:
These ones that only return "" - if they are not intended to be used, instead of returning an empty string, throw an UnsupportedOperationException

Reply:
Add throws message